frontend/js/routes.js

```diff
@@ -20,6 +20,7 @@ module.exports = (
   <Route path="/" component={Frame}>
     <IndexRoute component={Home} />
     <Route path="survey" component={SurveySelector} />
+    <Route path="survey/:id" component={Category} />
```

Comment:
we're using the same component here with no special logic within the component to handle id and without. is this intended?